src/app/courses-list/courses-list.component.spec.ts

```diff
@@ -1,25 +1,27 @@
-import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import {async, ComponentFixture, TestBed} from '@angular/core/testing';
 
-import { CoursesListComponent } from './courses-list.component';
+import {CoursesListComponent} from './courses-list.component';
+import {CUSTOM_ELEMENTS_SCHEMA} from "@angular/core";
 
 describe('CoursesListComponent', () => {
-  let component: CoursesListComponent;
-  let fixture: ComponentFixture<CoursesListComponent>;
+	let component: CoursesListComponent;
```

Comment:
You can test 
1) that every cours are present in the list.
2) a user can remove a course and the count of items will decrease